types/src/v0/nsproof.rs

```diff
@@ -0,0 +1,50 @@
+use hotshot_query_service::VidCommon;
+use hotshot_types::data::VidCommitment;
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    v0::{NamespaceId, NsIndex, NsPayload, NsTable, Payload, Transaction},
+    v0_1::ADVZNsProof,
+    v0_3::AvidMNsProof,
+};
+
+/// Each variant represents a specific minor version of a namespace proof.
```

Comment:
What does this "minor" refer to? I thought it's a pretty "major" change to the VID.

Reply:
rephrased in [97a33eb](https://github.com/EspressoSystems/espresso-network/pull/2866/commits/97a33eb2939a5d86c78b7543f1c9145c2a1897f5)